tests/test_auth.py

```diff
@@ -19,452 +19,451 @@
     add_role,
     add_user,
 )
-from tests import OAUTH_INFO_PROVIDER_LOGIN, TEST_USER_ID, TestCase
-
-
-class TestAuth(TestCase):
-    """Auth API tests"""
-
-    def test_nouser_logout(self):
-        """Confirm logout works without a valid user"""
-        response = self.client.get('/logout')
-        assert 302 == response.status_code
-
-    def test_local_user_add(self):
-        """Add a local user via flask_user forms"""
-        data = {
-            'password': 'one2Three',
-            'retype_password': 'one2Three',
-            'email': 'otu@example.com'}
-        response = self.client.post('/user/register', data=data)
-        assert response.status_code == 302
-        new_user = User.query.filter_by(username=data['email']).first()
-        assert new_user.active is True
-
-    def test_local_login_valid_username_and_password(self):
-        """login through the login form"""
-        # Create a user
-        email = 'localuser@test.com'
-        password = 'Password1'
-        user = self.add_user(
-            username='username',
-            email=email,
-            password=password
-        )
-
-        # Attempt to login with valid creds
-        response = self.local_login(user.email, password)
-
-        # Validate login was successful
+from tests import OAUTH_INFO_PROVIDER_LOGIN, TEST_USER_ID
+
+
+@pytest.fixture
+def test_auth_user(add_user):
+    # Create a user
+    email = 'localuser@test.com'
+    password = 'Password1'
+    user = add_user(
+        username='username',
+        email=email,
+        password=password
+    )
+    return user
+
+
+def test_nouser_logout(client, initialized_db):
+    """Confirm logout works without a valid user"""
+    response = client.get('/logout')
+    assert 302 == response.status_code
+
+
+def test_local_user_add(client):
+    """Add a local user via flask_user forms"""
+    data = {
+        'password': 'one2Three',
+        'retype_password': 'one2Three',
+        'email': 'otu@example.com'}
+    response = client.post('/user/register', data=data)
+    assert response.status_code == 302
+    new_user = User.query.filter_by(username=data['email']).first()
+    assert new_user.active is True
+
+
+def test_local_login_valid_username_and_password(test_auth_user, local_login):
+    test_auth_user = db.session.merge(test_auth_user)
+    """login through the login form"""
+    # Attempt to login with valid creds
+    response = local_login(test_auth_user.email, 'Password1')
+
+    # Validate login was successful
+    assert response.status_code is 200
```

Comment:
I typically only use the keyword `is` for comparison w/ `None`.  I think `==` makes more sense here.